src/components/Drawer/types.ts

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from 'react';
+
+export interface DrawerProps {
+  type?: 'left' | 'right';
```

Comment:
Instead writing `'left' | 'right'` everywhere, I think creating `DrawerPosition` type might be helpful.